MIGRATION_GUIDE.md

```diff
@@ -201,6 +213,20 @@ The additional parameters call **DESC DATABASE** (with `with_describe` turned on
 The outputs of both commands are held in `databases` entry, where **DESC DATABASE** is saved in the `describe_output` field, and **SHOW PARAMETERS IN DATABASE** in the `parameters` field.
 It's important to limit the records and calls to Snowflake to the minimum. That's why we recommend assessing which information you need from the data source and then providing strong filters and turning off additional fields for better plan performance.
 
+## v0.91.0 ➞ v0.92.0
```

Comment:
Shouldn't this be 92 -> 93?

Reply:
I moved this from the `CHANGES_BEFORE_V1` where it was like this, but looking at the commit history I realized that you are right. Moved to the proper section in 92->93.